server/utils/mpesa_utils.py

```
@@ -0,0 +1,278 @@
+                    'error': error_message,
+                    'response_code': response_data.get('ResponseCode')
+                }
+
+        except requests.exceptions.RequestException as e:
+            # Extract error message from response if available
+            error_msg = str(e)
+            if hasattr(e, 'response') and e.response is not None:
+                try:
+                    error_data = e.response.json()
+                    error_msg = error_data.get('errorMessage', error_data.get('errorMessage', str(e)))
+                except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)